File: unity_app/Assets/Scripts/Core/Enums/UIEnums.cs
namespace EventPlannerSim.Core
{
    /// <summary>
    /// Represents the apps available in the phone interface.
    /// </summary>
    public enum PhoneApp
    {
        Calendar,
        Messages,
        Bank,
        Contacts,
        Reviews,
        Tasks,
        Clients
    }

    /// <summary>
    /// Represents the current step in the tutorial sequence.
    /// </summary>
    public enum TutorialStep
    {
        Welcome,
        AcceptClient,
        SelectVenue,
        SelectCaterer,
        EventExecution,
        ViewResults,
        Complete
    }

    /// <summary>
    /// Represents background music tracks.
    /// </summary>
    public enum MusicTrack
    {
        MainMenu,
        Planning,
        Execution,
        Results,
        Celebration
    }

    /// <summary>
    /// Represents sound effects.
    /// </summary>
    public enum SoundEffect
    {
        ButtonClick,
        Success,
        Failure,
        Notification,
        CashRegister,
        Warning
    }
}

File: original-source/Models/Enums/UIEnums.swift
import Foundation

enum PhoneApp: String, CaseIterable, Codable, Hashable {
    case calendar
    case messages
    case bank
    case contacts
    case reviews
    case tasks
    case clients
}

enum TutorialStep: String, CaseIterable, Codable, Hashable {
    case welcome
    case acceptClient
    case selectVenue
    case selectCaterer
    case eventExecution
    case viewResults
    case complete
}

enum MusicTrack: String, CaseIterable, Codable, Hashable {
    case mainMenu
    case planning
    case execution
    case results
    case celebration
}

enum SoundEffect: String, CaseIterable, Codable, Hashable {
    case buttonClick
    case success
    case failure
    case notification
    case cashRegister
    case warning
}
